activesupport/lib/active_support/digest.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module ActiveSupport
+  class Digest
+    class <<self
+      attr_accessor :use_fips_approved_hash_function
+
+      def hexdigest(str)
+        new.hexdigest(str)
+      end
+    end
+
+    def initialize(fips_mode = false)
```

Comment:
Why not to use keyword arg. here?

Reply:
@simi: Not sure what you mean, could you clarify your question?

Reply:
`def initialize(fips_mode: false)` would make it much clearer to the caller what the boolean being given means.